### ISSUE TITLE
Issue with cisco.nxos.nxos_interfaces 'enabled' parameter on interfaces.

### ISSUE THREAD

**Issue by snehald12** [ansible-collections/cisco.nxos#662]
**ISSUE TYPE**

- Bug Report

**SUMMARY**

Facing an issue on the nxos_interfaces collection on playbook run it does not 'shut' or 'no shut' interfaces on certain NXOS devices.

The playbook is being executed from CLI with Ansible core 2.12.6 and cisco.nxos 4.1.0

Playbook:
~~~
---
- name: Interface Update Implementation
  hosts: all
  gather_facts: no
  tasks:
  - name: Update Interface Status
    cisco.nxos.nxos_interfaces:
      config:
      - name: Ethernet1/xx
        description: Interface_Description
        duplex: auto
        enabled: true
        ip_forward: false
        mode: layer3
        mtu: xxxx
        speed: '1000'
      - name: Ethernet1/xx
        description: Interface_Description
        duplex: auto
        enabled: true
        ip_forward: false
        mode: layer3
        mtu: xxxx
        speed: '1000'
      state: merged
~~~

Interface status before/after playbook run:
~~~
Before playbook run:
# sh running-config interface Ethernet 1/xx

!Command: show running-config interface Ethernet1/xx


version 7.0(3)I7(6) Bios:version 07.65 

interface Ethernet1/xx
  description Interface_Description
  mtu xxxx
  speed 1000

interface Ethernet1/xx
  description Interface_Description
  mtu xxxx
  speed 1000


After playbook run:
# sh running-config interface Ethernet 1/xx

!Command: show running-config interface Ethernet1/xx
!Running configuration last done at: Mon Apr  3 11:32:17 2023
!Time: Mon Apr  3 11:32:34 2023

version 7.0(3)I7(6) Bios:version 07.65 

interface Ethernet1/xx
  description Interface_Description
  mtu xxxx
  speed 1000

interface Ethernet1/xx
  description Interface_Description
  mtu xxxx
  speed 1000
~~~

**STEPS TO REPRODUCE**: N/A

**EXPECTED RESULTS**
After playbook runit should show the staus on interface depending on values passed on enabled parameter on playbook. For example if enabled is true the interface should show the no shutdown status on it.
 

**ACTUAL RESULTS**
It doesn't reflect any status "shutdown or no shutdown" on the interface.

**ADDITIONAL INFORMATION**
This is request from a customer.

Thanks.

**Comment by NilashishC** [comment]
@snehald12 could you please share the output of `show running-config all | incl 'system default switchport'` from the affected devices so that we can debug this further? Thank you.

**Comment by NilashishC** [comment]
@snehald12 Gentle reminder. In order to debug this further we would need to aforementioned information from you. Alternatively, if this issue is resolved now, please update this ticket and we'll close it. Thanks!

**Comment by NilashishC** [comment]
@snehald12 Since there's no activity on this issue, I'll be closing it for now. Note that a similar issue was reported in #749 and has been fixed with #813. The v6.0.2 release of this collection contains this patch. I recommend upgrading to the latest version and verifying if this issue still persists.

Thank you!
